compiler/rustc_mir/src/transform/copy_prop.rs

```diff
@@ -30,177 +29,151 @@ use rustc_middle::ty::TyCtxt;
 pub struct CopyPropagation;
 
 impl<'tcx> MirPass<'tcx> for CopyPropagation {
-    fn run_pass(&self, tcx: TyCtxt<'tcx>, _source: MirSource<'tcx>, body: &mut Body<'tcx>) {
+    fn run_pass(&self, tcx: TyCtxt<'tcx>, source: MirSource<'tcx>, body: &mut Body<'tcx>) {
         // We only run when the MIR optimization level is > 1.
         // This avoids a slow pass, and messing up debug info.
         if tcx.sess.opts.debugging_opts.mir_opt_level <= 1 {
```

Comment:
if you change this to

```suggestion
        if tcx.sess.opts.debugging_opts.mir_opt_level <= 0 {
```

we can perf-test this if you want

Reply:
Ok, lets try it. Later we could measure the old version for comparison.